webapp/portlet/src/main/webapp/vue-apps/version-history-drawer/components/VersionCard.vue

```diff
@@ -0,0 +1,235 @@
+      </div>
+      <div class="version-update-date">
+        <date-format
+          class="text-light-color text-truncate caption"
+          :value="versionDate"
+          :format="dateTimeFormat" />
+      </div>
+    </div>
+    <div class="mb-1 description-restore-wrapper d-flex justify-space-between pt-2">
+      <div
+        v-if="enableEditDescription"
+        class="item-version-description">
+        <a
+          v-if="!versionObject.summary && descriptionInputHidden"
+          class="descriptionContent position-relative dark-grey-color"
+          @click.stop.prevent="showInput">
+          {{ $t('versionHistory.description.add') }}
+        </a>
+        <v-progress-circular
+          v-if="isUpdatingDescription"
+          :size="20"
+          color="primary"
+          indeterminate />
+        <v-tooltip
+          v-else-if="descriptionInputHidden"
+          bottom>
+          <template #activator="{ on, attrs }">
+            <p
+              v-bind="attrs"
+              v-on="on"
+              class="descriptionContent pa-0 text-truncate position-relative dark-grey-color"
+              @click.stop.prevent="showInput">
+              {{ versionObject.summary }}
+            </p>
+          </template>
+          <div
+            class="caption tooltip-version text-break">
+            {{ versionObject.summary }}
+          </div>
+        </v-tooltip>
+        <v-text-field
+          v-if="canManage"
+          ref="NewDescriptionInput"
+          v-show="!descriptionInputHidden && !isUpdatingDescription"
+          v-model="newDescription"
+          :placeholder="$t('versionHistory.description.placeholder')"
+          class="description pa-0 dark-grey-color"
+          outlined
+          dense
+          autofocus
+          @click.stop.prevent
+          @keyup.enter="updateDescription">
+          <div slot="append" class="d-flex mt-1">
+            <v-icon
+              :class="descriptionMaxLengthReached && 'not-allowed' || 'clickable'"
+              :color="descriptionMaxLengthReached && 'grey--text' || 'primary'"
+              class="px-1"
+              small
+              @click.stop.prevent="updateDescription">
+              fa-check
+            </v-icon>
+            <v-icon
+              class="clickable px-0"
+              color="red"
+              small
+              @click.stop.prevent="resetInput">
+              fa-times
+            </v-icon>
+          </div>
+        </v-text-field>
+      </div>
+      <div
+        v-if="!versionObject.current && canManage && !disableRestoreVersion"
+        v-show="descriptionInputHidden"
+        class="item-version-restore">
+        <v-progress-circular
+          v-if="isRestoringVersion"
+          :size="20"
+          color="primary"
+          indeterminate />
+        <v-tooltip
+          v-else
+          bottom>
+          <template #activator="{ on, attrs }">
+            <v-icon
```

Comment:
```suggestion
        <v-tooltip bottom>
          <template #activator="{ on, attrs }">
            <v-btn icon :loading="isRestoringVersion" v-bind="attrs" v-on="on" @click.stop.prevent="restoreVersion" ...
            <v-icon
```
Use loading effect on `v-btn` instead of using a `v-progress-circular`